api/apps/articles/model.coffee

```diff
@@ -385,9 +387,40 @@ typecastIds = (article) ->
     biography_for_artist_id: ObjectId(article.biography_for_artist_id) if article.biography_for_artist_id
     super_article: if article.super_article?.related_articles then _.extend article.super_article, related_articles: article.super_article.related_articles.map(ObjectId) else {}
 
+sendArticleToSailthru = (article, cb) =>
+  images = {}
+  tags = article.keywords.concat ['article']
+  tags = tags.concat ['artsy-editorial'] if article.author_id is "503f86e462d56000020002cc"
+  tags = tags.concat ['magazine'] if article.featured is true
+  if article.email_metadata?.image_url
+    images =
+      full: url: crop(article.email_metadata.image_url, { width: 1200, height: 706 } )
+      thumb: url: crop(article.email_metadata.image_url, { width: 900, height: 530 } )
+  if NODE_ENV is 'production'
```

Comment:
Yeah I think this is the way to go then. Ideally there'd be a "staging" version of Sailthru and you'd set the keys above to be different. However, without that I think this is next best approach.